vm/rust/src/jsonrpc.rs

```diff
@@ -298,26 +322,28 @@ impl From<OrderedL2ToL1Message> for OrderedMessage {
 pub struct Retdata(pub Vec<StarkFelt>);
 
 fn make_state_diff(
-    state: &mut TransactionalState<JunoStateReader>,
+    state: &mut TransactionalState<CachedState<JunoStateReader>>,
     deprecated_declared_class: Option<ClassHash>,
```

Comment:
nite: can we rename this deprecated_declared_class_hash?

Reply:
Same for the deprecated_declared_class variable in new_transaction_trace?

Reply:
done, thanks